docs/TROUBLE.md

```diff
@@ -0,0 +1,175 @@
+
+This page attempts to provide a step by step guide to isolate and correct some common problems.  It covers only a core use case where an OpenTX or EdgeTX radio with a mLRS Tx module is used to communicate with a mLRS receiver connected to an ArduPilot flight controller to provide both RC control and MAVLink Telemetry to a ground station and/or [Yaapu Telemetry App](CRSF.md).  Users with other use cases in mind may still find some parts of this page helpful.
+
+Numbered topics here assume correct verification of previous functionality so don't skip ahead.  You must establish correct operation for each numbered topic in turn or the deductions we make later may not be correct!  Each numbered topic also assumes the state and configuration established in previous topics.  For example, topic 3 instructs you to power on both your receiver and Tx module and verify the connection.  All subsequent topics assume they are powered on and connected.
+
+Each numbered topic is structured as a test procedure with expected results followed by a list of things to check or tests to perform to help locate the problem if the expected test results are not observed.
+
+Note also that we don't mention things to check which were demonstrated to be correctly configured in previous topics.
+
+Some notation and terminology:
+
+- We use "Tx module" to refer to a mLRS radio module in the Tx or transmitter role.  In the use case assumed here, it will be installed in a OpenTX or EdgeTX radio handset.
+
+- We use "receiver" to refer to a mLRS radio module used in the receiver or Rx role.  In this use case, it will be mounted on the vehicle and connected to the flight controller.
+
+- We use lower case "rx" when refering to the receive pad or pin of a serial port connection.
+
+- We use lower case "tx" when refering to the transmit pad or pin of a serial port connection.
+
+## 1. Confirm successful firmware install
+
+In most cases, you can confirm that mLRS firmware is installed by observing the LED.  When either an receiver or Tx module is powered on by itself, a LED (usually red) should flash at 2 Hz (twice per second, on for 1/4 second then off for 1/4 second repeatedly).
+
+- If you don't see any LED activity, check power connections and voltage
+- If you don't see the expected LED pattern, follow the mLRS firmware install [documentation](https://github.com/olliw42/mLRS-docu?tab=readme-ov-file#stm32-hardware) for your hardware.
+- If you have trouble reliably flashing via ST-Link, try shorter wires, 20cm or less.
+
+## 2. Confirm operation of the mLRS LUA script
+
+Carefully install your Tx module into your OpenTX or EdgeTX radio.  Some Tx modules can be fussy on pin alignment;  Don't use excessive force or you may break something.  Follow the [instructions](LUA.md) to configure your radio and install the mLRS LUA script. Then, confirm you can talk to the Tx module via the LUA script.  You should see the TX module firmware version reported at the top of the screen.
```

Comment:
'alignment; don't use' instead of 'alignment;  Don't use'

Reply:
Damn, 45 years of habit is hard to break.